### ISSUE TITLE
Test examples for angular2-go?

### ISSUE THREAD

**Issue by jesperronn** [johnpapa/angular2-go#9]
Hi just a heads-up, currently quite a few repositories provide an example NG2 app ready for use.

Some have tests, some have sass, have other features.

I am currently evaluating some different projects (starters) I can see that tests will help me further along in my projects.

What are your directions for this example project? Would you want testing, continuous integration or similar to be added to this project?

**Comment by johnpapa** [comment]
yep. there are other repos for starter kits in here too. Perhaps add some to the there-and-back-again repo in here.

**Comment by antonybudianto** [comment]
@jesperronn , take a look at my [starter](https://github.com/antonybudianto/angular2-starter) got SASS and unit testing with Gulp workflow
